### ISSUE TITLE
Intl.PluralRules.~locale.fr not working correctly

### ISSUE THREAD

**Issue by aronwoost** [Financial-Times/polyfill-library#973]
Bug

## What

`Intl.PluralRules.~locale.fr` polyfill is not working correctly (tested on IE11 and Safari 12).

## Details

How to reproduce:

```html
<!DOCTYPE html>
<html>
<head>
  <title></title>
  <script src="https://polyfill.io/v3/polyfill.js?version=3.101.0&flags=gated&features=Intl.PluralRules.~locale.fr"></script>
  <script type="text/javascript">
    console.log(new Intl.PluralRules('fr').select(0));
    console.log(new Intl.PluralRules('fr').select(1));
    console.log(new Intl.PluralRules('fr').select(2));
  </script>
</head>
<body>
  test
</body>
</html>
```

`console.log(new Intl.PluralRules('fr').select(2));` is throwing with:

![image](https://user-images.githubusercontent.com/408962/104367396-087fc000-551b-11eb-8613-54d0b22f0be4.png)

It should log `"others"`.

From reading the code I think that `e` is indeed never defined. 🤔

**However**, what does work is this:

```html
<!DOCTYPE html>
<html>
<head>
  <title></title>
  <script src="https://polyfill.io/v3/polyfill.js?version=3.101.0&flags=gated&features=Intl.PluralRules"></script>
  <script src="https://unpkg.com/@formatjs/intl-pluralrules@4.0.5/locale-data/fr.js"></script>
  <script type="text/javascript">
    console.log(new Intl.PluralRules('fr').select(0));
    console.log(new Intl.PluralRules('fr').select(1));
    console.log(new Intl.PluralRules('fr').select(2));
  </script>
</head>
<body>
  test
</body>
</html>
```

It does correctly output:

![image](https://user-images.githubusercontent.com/408962/104367769-90fe6080-551b-11eb-8ffa-8962e445e775.png)

So I suspect something went wrong there with coping the polyfill.

**Comment by JakeChampion** [comment]
Thanks for reporting this @aronwoost.

@longlho, this looks like an issue within the bundled version of the PluralRules polyfill, would you be able to look into this?

**Comment by longlho** [comment]
dupe of https://github.com/formatjs/formatjs/issues/2482

we fixed it in 4.0.5 so just need an upgrade

**Comment by longlho** [comment]
@JakeChampion looks like this already has the fix. Maybe u need to cut a release?

**Comment by longlho** [comment]
yeah see comment in https://github.com/Financial-Times/polyfill-library/pull/967

**Comment by JakeChampion** [comment]
Thanks for noticing the missed release. I've now released this update in [v3.102.0](https://github.com/Financial-Times/polyfill-library/releases/tag/v3.102.0)

**Comment by aronwoost** [comment]
You're welcome and thanks for releasing! 💪🏼

Pls let me know when `polyfill-service` is updated.
